docker/test/integration/cluster/checkers/PrometheusChecker.py

```diff
@@ -60,16 +60,22 @@ def verify_queue_metrics(self):
     def verify_general_processor_metrics(self, metric_class, processor_name):
         labels = {'processor_name': processor_name}
         return self.verify_metrics_exist(['minifi_average_onTrigger_runtime_milliseconds', 'minifi_last_onTrigger_runtime_milliseconds',
-                                          'minifi_average_session_commit_runtime_milliseconds', 'minifi_last_session_commit_runtime_milliseconds'], metric_class, labels) and \
-            self.verify_metrics_larger_than_zero(['minifi_onTrigger_invocations', 'minifi_transferred_flow_files', 'minifi_transferred_to_success', 'minifi_transferred_bytes'], metric_class, labels)
+                                          'minifi_average_session_commit_runtime_milliseconds', 'minifi_last_session_commit_runtime_milliseconds',
+                                          'minifi_incoming_flow_files', 'minifi_incoming_bytes', 'minifi_bytes_read', 'minifi_bytes_written',
+                                          'minifi_processing_nanos'], metric_class, labels) and \
+            self.verify_metrics_larger_than_zero(['minifi_onTrigger_invocations', 'minifi_transferred_flow_files', 'minifi_transferred_to_success',
+                                                  'minifi_transferred_bytes', 'minifi_processing_nanos'],
```

Comment:
'minifi_processing_nanos' is in both checks -- is that intentional / useful?

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1909/commits/3cb95dab8b2fac7eb909ded2cd0ddd0d6436cff9